public_html/app/Models/AbstractModel.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace VFramework\Models;
+
+ use VFramework\Libraries\Database;
+
+ abstract class AbstractModel
+ {
+     /**
+      * @var Database
+      */
+     protected $db;
+
+     // Instantiates 'Database' class
+     public function __construct()
+     {
+         $this->db = new Database();
+     }
+
+
+     /**
+      * @param array $data
+      */
+     public function add($data)
+     {
+         unset($data['confirm_password']);
```

Comment:
For what reasons you unseting confirm_password here? This is abstract layer, you should avoid logic like this in the abstract layers.